.github/workflows/bundle-size-tracker.yml

```diff
@@ -0,0 +1,92 @@
+name: Bundle Size Tracker
+
+on:
+  pull_request:
+    branches:
+      - main
+      - releases/*
+      - features/*
+    paths:
+      - 'packages/fast-element/**'
+      - '.github/workflows/bundle-size-tracker.yml'
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  track-bundle-size:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout PR branch
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          cache: 'npm'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build packages
+        run: npm run build
```

Comment:
nit: this builds all packages for the entire repo (including the docs site), which is likely not what we want